docs/learn/tour/tables.md

```diff
@@ -1,209 +1,307 @@
 ---
-title: Tables in kdb+ | A whirlwind tour of kdb+ and q | Documentation for kdb+ and the q programming language
-author: Stephen Taylor
-date: February 2020
-description: How to construct, manipulate and save tables in kdb+
+title: Tables in kdb+ 
+author: Conor McCarthy
+date: December 2024
+description: A brief tour of tables in kdb+/q
 ---
-# Tables
+# Tables in kdb+
 
+This page introduces you to the types of tables which exist in q, how these tables are created and what makes tables in q so powerful.
 
+Unlike in many other programming languages where the tables you encounter are added as a second-class extension to the language, in q they are first-class objects. 
 
-Tables are first-class objects in q. 
+There are 5 types of table that you'll encounter as you learn more about kdb+:
 
+1. Table
+1. Keyed Table 
+1. Splayed Table
+1. Partitioned Table
+1. Segmented Table
 
-## Construct
+When dealing with these forms of tables there is a distinction between in-memory and on-disk datasets. We will deal with these two cases separately.
 
-Construct a small table using table notation.
+## In-Memory Tables
 
-```q
-q)ec1:([]city:`Istanbul`Moscow`London`StPetersburg;country:`Turkey`Russia`UK`Russia;pop:15067724 12615279 9126366 5383890)
-q)ec1
-city         country pop
------------------------------
-Istanbul     Turkey  15067724
-Moscow       Russia  12615279
-London       UK      9126366
-StPetersburg Russia  5383890
-```
+In-memory tables live RAM of a process. This memory is volatile and limited in size but allows for fast data retrieval. Typically you will be in a position to generate and access a maximum of 10s of gigabytes of data. As the data stored in RAM is volatile until the data is persisted any exiting of a process will result in the data being lost and will require the data to be regenerated, a task which may be complex.
 
-!!! tip "Equals means equals"
+### What is a Table?
 
-    In q names are assigned values with the colon.
-    The equals sign `=` is the Equals operator. It returns a boolean.
+At its most basic a table is a mapping between a list of column names which each have an associated list of corresponding column values. This column to list mapping form the basis behind why q tables are described as being column-oriented, this is in contrast to row-oriented tables in most relational databases. Additionally because lists are ordered so are the content of a column.
```

Comment:
```suggestion
A table is essentially a mapping between column names and their corresponding lists of values. This column-to-list mapping is why q tables are considered column-oriented, unlike the row-oriented tables in most relational databases. Additionally, since lists are ordered, the contents of each column are also ordered.
```